File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans = 0
        c = 0
        for i in nums:
            if i == 1:
                c += 1
                ans = max(c,ans)
                print(c)
            else:
                c = 0
                print(c,ans)
        return ans

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int sum = 0;
        int i = 0;
        int c = 0;
        while(i < nums.length){
            if(nums[i] != 1){
                if(sum < c){
                    sum = c;
                }
                c = 0;
            }else{
                c++;
            }
            i++;
        }
        if(sum < c){
            sum = c;
        }
        return sum;
    }
}